src/components/avatar/Avatar.types.tsx

```diff
@@ -0,0 +1,18 @@
+import { ViewProps } from 'react-native'
```

Comment:
q: why do you prefer to write the types in a different file? In my company we do it in the same file (easier to find/navigate). Naturally I'm biased, but would like to know your perspective.

Reply:
Ah forgot to link, I'm also biased [because of this article](https://www.totaltypescript.com/where-to-put-your-types-in-application-code)

Reply:
Hum...I was used to having them in the same file before as well but in my current company they are often separated and, despite making it a bit wonky sometimes to actually work with them, the upside is that the component file itself is "cleaner". I often see myself `Pick`/`Omit`ing some other component prop which then means that the type is not single use as well, I'm not sure what's the best way to do this either tbh

Reply:
But, for example, inlining types just feels very messy to me

Reply:
You could do what Matt suggests and create a components.types.ts but it would just mean having a longer file (with your component types still separated from your component which I feel would be worse honestly). I kinda like having very small files that serve a single purpose, it feels way easier to understand what they are about. I'm fine with keeping the types inside the components though, specially if we have a component per file

Reply:
I agree with inlining being very messy, and I'm also against "buckets" of files (eg `components.types.ts`). Not because the file gets huge but because it's hard to find stuff to forget deleting unused types.

Well, this project is an opportunity for us to try new things (and for me to explore different patterns), so it's your call. I'm good either way!